Sync/Synchronizers/Bookmarks/BookmarksSynchronizer.swift

```diff
@@ -118,34 +118,37 @@ open class BufferingBookmarksSynchronizer: TimestampedSingleCollectionSynchroniz
 
         statsSession.start()
         if !AppConstants.shouldMergeBookmarks {
-            if case .release = AppConstants.BuildChannel {
-                // On release, just mirror; don't validate.
-                run = doMirror
-            } else {
-                run = doMirror >>== effect({ result in
-                    // Just validate to report statistics.
-                    if case .completed = result {
-                        log.debug("Validating completed buffer download.")
-                        let _ = buffer.validate()
+            run = doMirror >>== { result in
```

Comment:
Cool thanks! I've already included a `took` field in the validation stats to tell us how long each validation will take:

https://github.com/mozilla-mobile/firefox-ios/pull/2734/files#diff-1a81823fc27dd5019ba11af19920296fR807